gemini/sample-apps/e2e-gen-ai-app-starter-pack/deployment/README.md

```diff
@@ -43,6 +43,13 @@ The application leverages [**Terraform**](http://terraform.io) to define and pro
    gcloud services enable serviceusage.googleapis.com cloudresourcemanager.googleapis.com cloudbuild.googleapis.com secretmanager.googleapis.com
    ```
 
+4. Enable required APIs in the Staging and Prod projects. This will be required for the Terraform deployment:
```

Comment:
terraform should enable APIs.

There is a special case for that as the project where you are running terraform needs some apis enabled to not fail but this won't be needed